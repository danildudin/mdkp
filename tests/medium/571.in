30 7
596 761 856 698 576 693 925 623 813 591 537 937 532 454 327 635 835 258 320 902 414 477 721 984 209 904 413 128 35 495 
994 180 214 979 61 191 413 378 193 153 782 779 694 629 614 465 960 575 553 478 118 581 470 175 694 997 213 140 737 735 6191
344 323 439 390 59 870 28 15 811 100 174 855 665 457 643 198 118 442 500 623 846 541 541 903 563 153 591 577 213 709 5584
524 894 983 280 12 715 942 864 99 140 927 898 85 441 141 241 366 578 323 793 119 198 576 239 677 339 12 98 649 829 5867
587 852 267 533 821 707 582 938 365 74 779 359 231 993 383 334 817 228 409 712 894 45 242 73 923 916 669 56 282 951 6063
404 784 237 977 142 860 486 747 889 401 522 244 755 45 494 340 874 344 292 919 164 783 588 898 62 746 977 616 108 742 5425
530 816 571 982 598 258 988 677 449 331 911 155 507 159 759 559 103 866 485 801 397 874 106 372 490 494 326 160 615 403 5369
116 795 15 23 654 211 710 832 420 204 173 16 171 450 906 723 615 42 725 533 879 311 788 598 460 28 349 505 768 555 6633
