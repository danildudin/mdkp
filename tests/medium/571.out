8992
12
0 2 4 6 8 9 11 13 15 22 23 25 
